docs/content/features/exploiters/smb.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "SMB Exploiter"
+draft: false
+description: "Exploits SMB"
+tags: ["exploiter", "smb", "brute force"]
+pre: "<i class='fa fa-folder-open'></i> "
+---
+
+## SMB
+
+[SMB (Server Message Block)](https://en.wikipedia.org/wiki/Server_Message_Block)
+is a network protocol primarily used in Windows for inter-process communication,
+and providing shared access to files, printers, and serial ports between nodes
+on a network.
```

Comment:
Maybe some information about how/why this allows you to execute commands would be good.